python/lsst/ts/m2gui/controltab/tab_utility_view.py

```diff
@@ -33,7 +39,424 @@ class TabUtilityView(TabDefault):
+
+        sensors_theta = self.model.utility_monitor.get_displacement_sensors(
+            DisplacementSensorDirection.Theta
+        )
+        sensors_delta = self.model.utility_monitor.get_displacement_sensors(
+            DisplacementSensorDirection.Delta
+        )
+
+        for sensor_theta, sensor_delta in zip(sensors_theta, sensors_delta):
+            layout.addRow(sensor_theta + ":", self._displacements[sensor_theta])
+            layout.addRow(sensor_delta + ":", self._displacements[sensor_delta])
+            self._add_empty_row_to_layout(layout)
+
+        return create_group_box("Displacement Sensors", layout)
+
+    def _set_signal_utility(self, signal_utility):
+        """Set the utility signal with callback functions. This signal provides
+        the utility status contains the power, inclinometer, breaker,
+        temperature, displacements, etc.
+
+        Parameters
+        ----------
+        signal_utility : `SignalUtility`
+            Signal of the utility.
+        """
+
+        signal_utility.power_motor.connect(self._callback_power_motor)
+        signal_utility.power_communication.connect(self._callback_power_communication)
+
+        signal_utility.inclinometer.connect(self._callback_inclinometer)
+
+        signal_utility.breaker_status.connect(self._callback_breakers)
+
+        signal_utility.temperatures.connect(self._callback_temperatures)
+
+        signal_utility.displacements.connect(self._callback_displacements)
+
+    @Slot()
+    def _callback_power_motor(self, power_motor):
+        """Callback of the utility signal for the motor power.
+
+        Parameters
+        ----------
+        power_motor : `tuple`
+            Motor power: (voltage, current). The data type is float. The units
+            are volt and ampere respectively.
+        """
+
+        self._power_inclinometer["power_voltage_motor"].setText(f"{power_motor[0]} V")
+        self._power_inclinometer["power_current_motor"].setText(f"{power_motor[1]} A")
+
+    @Slot()
+    def _callback_power_communication(self, power_communication):
+        """Callback of the utility signal for the communication power.
+
+        Parameters
+        ----------
+        power_communication : `tuple`
+            Communication power: (voltage, current). The data type is float.
+            The units are volt and ampere respectively.
+        """
+
+        self._power_inclinometer["power_voltage_communication"].setText(
+            f"{power_communication[0]} V"
+        )
+        self._power_inclinometer["power_current_communication"].setText(
+            f"{power_communication[1]} A"
+        )
+
+    @Slot()
+    def _callback_inclinometer(self, inclinometer):
+        """Callback of the utility signal for the inclinometer angle.
+
+        Parameters
+        ----------
+        inclinometer : `float`
+            Inclinometer angle in degree.
+        """
+        self._power_inclinometer["inclinometer"].setText(f"{inclinometer} degree")
```

Comment:
This would be much better handled with a subclass of QLabel, see CustomLabels in ts_cRIOpy/python/lsst/cRIOpy/GUI/CustomLabels.py, where you can create a class with astropy.unit argument - this allows e.g. change units if needed by a signal, etc..but your choice.

Reply:
I am not sure it is a good idea to have another dependency of `astropy` or not for the unit. I think to label the unit correctly is good enough at this moment. Thanks!